projectm_sys/.github/workflows/build_osx.yml

```diff
@@ -0,0 +1,47 @@
+name: macOS (x86_64)
```

Comment:
it's possible to use a "matrix" of configs for reusing a workflow on different configs